scripts/prepare_enclave_metadata.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
This file could be better named, I think. Maybe something like save_enclave_id_to_admin.sh

Reply:
Updated